packages/react-components/react-accordion/src/components/Accordion/Accordion.types.ts

```diff
@@ -49,6 +49,7 @@ export type AccordionProps<Value = AccordionItemValue> = ComponentProps<Accordio
   /**
    * Callback to be called when the opened items change.
    */
+  // eslint-disable-next-line @fluentui/consistent-callback-type -- callback should be typed with EventHandler, but we can't break existing callbacks
```

Comment:
_super-nit_ on wording for these comments, to be more concise and not mention breaking things:
```suggestion
  // eslint-disable-next-line @fluentui/consistent-callback-type -- can't change type of existing callback
```

Reply:
Updated for every comments